kernel/kernel-idgenerator-service/src/main/java/io/mosip/kernel/idgenerator/verticle/HttpServerVerticle.java

```diff
@@ -100,4 +105,17 @@ public void start(Future<Void> future) {
 			}
 		});
 	}
+
+	private void accessLogHandler(final RoutingContext context, AccessLogHandler accessLogHandler) {
```

Comment:
Rename this method to addAccessLogHandler and remove unwanted line brakes inside the method

Reply:
done